app/actions/websocket/index.ts

```diff
@@ -153,14 +136,14 @@ async function doReconnect(serverUrl: string) {
 
     const {database} = operator;
 
-    const lastDisconnectedAt = await getWebSocketLastDisconnected(database);
-    resetWebSocketLastDisconnected(operator);
+    const lastFullSync = await getLastFullSync(database);
```

Comment:
How often do we expect a full sync to happen if the connection is stable? I like the idea of using a timestamp where we know to have correct data, but I'm wondering if this might cause problems if the WS disconnects after having been connected for a longer period of time

Reply:
Any time we open the app fresh, we perform a full sync. In general, we only get one full refresh each session. But this approach tries to avoid problems with the websocket closing too late, and important changes happening on that time delta. Shouldn't happen often, but I prefer to be careful with that.